upgrade_guide.md

```diff
@@ -25,3 +25,23 @@
 - Get your developer api key from the [dashboard](https://stardust.neogoma.com/profile).
 - Fill the API key into the __Stardust SDK__ script of the **StardustComponents** prefab.
 - For better naming conventions **ALL** Uunity events have been renamed with the "on" prefix for example __dataCapturedSucessfully__ has been renamed to __onDataCapturedSucessfully__. Just rename all your event refrences with the prefix "on"
+
+## Updating with a newly released package
+
+In order to update your sdk to a newer release version all you have to do is download the new release version package and import it into unity. You don't need to delete the previous package as Unity replaces only the files that changed.
+
+### Downloading the latest release package
+
+- Go over to our SDK repository [releases](https://github.com/Neogoma/stardust-SDK/releases).
```

Comment:
Change the link to https://github.com/Neogoma/stardust-SDK/releases/latest

It will automatically point at the latest version

Reply:
Changed it 👍